src/protocols/interaction_model/Constants.h

```diff
@@ -62,6 +62,49 @@ enum class MsgType : uint8_t
     TimedRequest          = 0x0a,
 };
 
+//TODO: Cleanup Reserved protocolCode from ember
+enum class ProtocolCode : uint16_t
+{
+    kSuccess                = 0x0,
+    kFailure                = 0x01,
+    kInvalidSubscription    = 0x7d,
+    kUnsupportedAccess      = 0x7e,
+    kUnsupportedEndpoint    = 0x7f,
+    kInvalidAction          = 0x80,
+    kUnsupportedCommand     = 0x81,
+    kReservedUnsuptedGeneralCommand   = 0x82,
+    kReservedUnsupManufClusterCommand = 0x83,
+    kReservedUnsupManufGeneralCommand = 0x84,
```

Comment:
If we can avoid assigning any semantically meaningful names to the reserved values, that would be ideal.  So `Reserved1`, `Reserved2`, etc.  Or if desired `Reserved82 = 0x82` and so forth. Or just not add them to the enum at all.

Reply:
Done, thanks